app/helpers/host_description_helper.rb

```diff
@@ -0,0 +1,140 @@
+module HostDescriptionHelper
+  UI.register_host_description do
+    multiple_actions_provider :base_multiple_actions
+    overview_fields_provider :base_status_overview_fields
+    overview_fields_provider :base_host_overview_fields
```

Comment:
I do not think you can call the same method twice in one block ( == twice for one UI::HostDescription instance) as it re-registers the provider method. In this case `:base_status_overview_fields` is overwritten by `:base_host_overview_fields`.

Reply:
Improved method definition, now it should allow multiple calls.